matchms/Spectrum.py

```diff
@@ -33,6 +33,14 @@ class Spectrum:
         0.7
         spectrum1
 
+    To visually inspect a spectrum run ``spectrum.plot()``
```

Comment:
Can you move this code to the docstring of the plot() method

Reply:
True...Done.